qiskit/tools/jupyter/version_table.py

```diff
@@ -38,25 +38,9 @@ def qiskit_version_table(self, line="", cell=None):
         packages = []
         qver = qiskit.__qiskit_version__
 
-        packages.append(("Qiskit", qver["qiskit"]))
-        if qver["qiskit-terra"]:
-            packages.append(("Terra", qver["qiskit-terra"]))
-        if qver["qiskit-aer"]:
-            packages.append(("Aer", qver["qiskit-aer"]))
-        if qver["qiskit-ignis"]:
-            packages.append(("Ignis", qver["qiskit-ignis"]))
-        if qver["qiskit-aqua"]:
-            packages.append(("Aqua", qver["qiskit-aqua"]))
-        if qver["qiskit-optimization"]:
-            packages.append(("Optimization", qver["qiskit-optimization"]))
-        if qver["qiskit-machine-learning"]:
-            packages.append(("Machine Learning", qver["qiskit-machine-learning"]))
-        if qver["qiskit-nature"]:
-            packages.append(("Nature", qver["qiskit-nature"]))
-        if qver["qiskit-finance"]:
-            packages.append(("Finance", qver["qiskit-finance"]))
-        if qver["qiskit-ibmq-provider"]:
-            packages.append(("IBM Q Provider", qver["qiskit-ibmq-provider"]))
+        for pkg in qver:
+            if qver[pkg]:
```

Comment:
This will exclude the metapackage if it's not installed, your earlier revisions left the metapackage line intact even if it was none. Did you want to leave it?

Reply:
Oops, I did this on my phone and left a review on the wrong PR. I left the same comment in the parent PR: https://github.com/Qiskit/qiskit-terra/pull/6589#discussion_r657250484